src/metrics/reef_indices.jl

```diff
@@ -0,0 +1,243 @@
+using FLoops, DataStructures
+
+"""
```

Comment:
Are there any references for this? All good if not

Reply:
Not yet I don't think but I'll ask